src/foam/nanos/jetty/ThreadpoolConfig.js

```diff
@@ -0,0 +1,27 @@
+/**
+* @license
+* Copyright 2020 The FOAM Authors. All Rights Reserved.
+* http://www.apache.org/licenses/LICENSE-2.0
+*/
+foam.CLASS({
+  package: 'foam.nanos.jetty',
+  name: 'ThreadpoolConfig',
```

Comment:
Could we rename this JettyThreadPoolConfig so it isn't confused with the NANOS threadpool?

Reply:
Yes, that is in my notes above.

Reply:
the class itself is already in the jetty package. the renaming was at the service (nspec) level.